Resources/Prototypes/Catalog/Fills/Crates/security.yml

```diff
@@ -33,15 +33,6 @@
 #      - Pepperspray
 #      - GrenadeTeargas
 
-- type: entity
-  id: CrateSecurityLaser
-  parent: CrateSecgear
-  components:
-  - type: StorageFill
-    contents:
-      - id: WeaponLaserGun
-        amount: 3
-
```

Comment:
removing this breaks the CentCom map, which is partly why tests failed

Reply:
Ah, suppose that makes sense, my main goal was it make sure it was organized, however I guess I could still return it to the security fill file and it still function.

Reply:
It tries to add a CrateSecurityLaser I believe, so either the map would need to be updated or there needs to be a CrateSecurityLaser

Reply:
Hmm, I'm leaning more towards a map change however I also recognize it'd be easier to change it on my end.

Reply:
I'll talk with the mapper later today and see if we can work something out.

Reply:
I dont think it should be hard to change yourself on the map, (altho I've never done it) - for me, its line 51016 of centcomm.yml in Maps which says type: CrateSecurityLaser, which is prob all that needs to be changed.

Reply:
You can just delete the relevant yml node and it will work as long as nothing else references it (as a crate I doubt anything would).

Reply:
Alrighty, that should have fixed it.